mcs/class/System/Test/System.Net/DnsTest.cs

```diff
@@ -205,12 +206,21 @@ public void GetHostByName ()
 			}
 		}
 
+		static IPAddress GetNonIPv6Address (IPHostEntry h)
+		{
+			var al0 = h.AddressList [0];

```

Comment:
You're right of course, that was wishful thinking on my part :) I don't think we test in v6 networks right now.